src/less/notice/ds6/notice.less

```diff
@@ -6,43 +6,53 @@
 @import "../base/notice.less";
 
 // stylelint-disable no-descending-specificity
+@page-notice-information-border-color: @color-b4;
+@page-notice-information-background-color: #f1f8fe;
+@page-notice-confirmation-border-color: @color-g4;
+@page-notice-confirmation-background-color: #effef0;
+@page-notice-attention-border-color: @color-r4;
+@page-notice-attention-background-color: #fff5f8;
+@page-notice-cta-button-color: @color-b4;
+@page-notice-cta-border-color: @color-b4;
 
 .page-notice {
-    background-color: @color-white;
     border-style: solid;
-    border-width: 4px;
+    border-width: 1px;
+    flex-wrap: wrap;
     font-size: @font-size-14;
     margin: 8px 0;
 }
 
 .page-notice--confirmation {
-    border-color: @color-g4;
+    background-color: @page-notice-confirmation-background-color;
+    border-color: @page-notice-confirmation-border-color;
 }
 
 .page-notice--attention {
-    border-color: @color-r4;
+    background-color: @page-notice-attention-background-color;
+    border-color: @page-notice-attention-border-color;
 }
 
 .page-notice--information {
-    border-color: @color-b4;
+    background-color: @page-notice-information-background-color;
+    border-color: @page-notice-information-border-color;
 }
 
 .page-notice__status {
     align-items: center;
     color: @color-white;
-    padding: 12px 16px 12px 12px;
+    padding: 12px 17px 12px 17px;
 }
 
-.page-notice__content p {
-    margin: 0 16px 0 0;
-
-    &:first-child {
-        margin-top: 16px;
-    }
+.page-notice__content {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin: 17px 17px 17px 0;
```

Comment:
What's the reasoning to go to 17px? Seems like that is an odd padding given most specs are bases of 8px (16px being 2 x 8px).

Reply:
This is the size that design had on their mockups. Im guessing it was an error. I'll double check